From d14ae80cd10713a8754e273bc57f377335351b29 Mon Sep 17 00:00:00 2001
From: Fabien Dessenne <fabien.dessenne@foss.st.com>
Date: Mon, 18 Oct 2021 12:55:33 +0200
Subject: [PATCH 304/529] ARM: configs: enable STM32 IPCC in multi_v7 addons
 fragment

This patch adds fragments for the STM32 IPCC driver to multi_v7 addons.

Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/283665
Signed-off-by: Fabien Dessenne <fabien.dessenne@foss.st.com>
Change-Id: I8e35e8957f194ef9695f1742c50542450b1c2cd0
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -48,6 +48,7 @@ CONFIG_SCHED_MC=y
 CONFIG_SENSORS_IIO_HWMON=y
 CONFIG_SERIAL_NONSTANDARD=y
 CONFIG_SRAM=y
+CONFIG_STM32_IPCC=y
 CONFIG_TEE=y
 CONFIG_THERMAL=y
 CONFIG_TMPFS=y
